routes/organisation/steps.js

```diff
@@ -73,7 +74,13 @@ const internalRoutes = {
     controller: TeamCreateController,
     fields: ['teamName', 'permissionCheckboxes'],
     entryPoint: true,
-    template: '/add-team.html'
+    template: '/add-edit-team.html'
+  },
+  '/:orgId/team/:teamId/edit': {
```

Comment:
previously I had a bit of a problem with assigning `/edit` as HMPO form wizard has default edit route. just double check if it is okay ? I had to make `/modify` rather than `/edit` but if it works then all good

Reply:
Yeah I recall something about that but all is working at least locally. I'll leave and test in envs.